modules/installation-osp-kuryr-config-yaml.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-kuryr-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for OpenStack with Kuryr SDN
+
+To deploy with Kuryr SDN instead of the default OpenShift SDN, you simply need
+to modify the `install-config.yaml` file to include `Kuryr` as the desired
+`networking.networkType` and proceed with the same steps as with the default
```

Comment:
s/proceed with the same steps as with the default OpenShift SDN/proceed with the default OpenShift SDN steps